setup.py

```diff
@@ -12,7 +12,7 @@
     setup(
         name="pg_es_fdw",
         packages=["pg_es_fdw"],
-        version="0.8.0",
+        version="0.9.0",
```

Comment:
I use git flow for managing releases so the version bump will be in a separate commit.